### PR TITLE
Nav V2

### PR DESCRIPTION
**Pull request by imranolas**
- [x] Top level nav styling
- [x] Desktop nav layout
- [x] Mobile nav layout/behaviour
- [x] Mobile styles
- [x] Membership links
- [x] No-js

Dependant on https://github.com/Financial-Times/next-navigation/pull/57

### CODE REVIEW

File: src/scss/header-v2/_navigation.scss [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+@include oColorsSetColor('nav-warm-1', #e6ded4);
+
+@include oColorsSetUseCase(nav, text, 'white');
+@include oColorsSetUseCase(nav, background, 'grey-tint5');
+@include oColorsSetUseCase(link-hover, text, 'blue-2');
+@include oColorsSetUseCase(sub-nav, text, 'nav-warm-1');
+@include oColorsSetUseCase(sub-nav-heading, background, 'nav-warm');
+@include oColorsSetUseCase(sub-nav, background, 'cold-2');
+@include oColorsSetUseCase(sub-nav-hover, background, 'nav-warm-1');
+@include oColorsSetUseCase(header, border, 'grey-tint3');
```
Comment (kaelig) [inline src/scss/header-v2/_navigation.scss]: Great use of o-colors!

File: src/scss/header-v2/_navigation.scss [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+.next-navigation-v2,
+.next-navigation-v2__group__items {
+	list-style: none;
+	margin: 0;
+	padding: 0;
+	height: 100%;
+	background: linear-gradient(90deg, oColorsGetPaletteColor('cold-2') 41.667%, oColorsGetPaletteColor('nav-warm-1') 41.667%);
+	@include oGridRespondTo(M) {
+		background: oColorsGetPaletteColor('cold-2');
+	};
```
Comment (kaelig) [inline src/scss/header-v2/_navigation.scss]: This notation `};` works but I'd expect it to break easily with a Sass update.

File: src/scss/header-v2/_navigation.scss [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+
+	@include oGridRespondTo(M) {
+		@include oGridResetColumn;
+		padding-left: 20px;
+	}
+	@include oGridRespondTo($until: M) {
+		color: oColorsGetPaletteColor('white');
+		text-align: right;
+
+		&:hover {
```
Comment (kaelig) [inline src/scss/header-v2/_navigation.scss]: What about the focus state? Is this not focusable?

File: src/scss/header-v2/_navigation.scss (hunk 2 above)
Comment (imranolas) [inline src/scss/header-v2/_navigation.scss]: That's a typo. Good catch!

File: src/scss/header-v2/_navigation.scss (hunk 3 above)
Comment (imranolas) [inline src/scss/header-v2/_navigation.scss]: Good point. Thanks

**Pull request merged**